Lib/test/test_shutil.py

```diff
@@ -33,6 +33,29 @@
 except ImportError:
     UID_GID_SUPPORT = False
 
+
+def _has_functional_lchmod():
+    if not support.can_symlink() or not hasattr(os, 'lchmod'):
+        return False
+
+    fname = tempfile.mktemp()
```

Comment:
Should this be `mkstemp`? `mktemp` [seems to be deprecated](https://docs.python.org/2/library/tempfile.html#tempfile.mktemp).

Reply:
This was copied from adjacent test (in the original PR).  It's not exactly that `mktemp` is deprecated, it's that it's _insecure_.  However in this case, the same security problem would exist since it's essentially:

1. `mkstemp`
2. unlink the file made by `mkstemp`
3. symlink to the location made by `mkstemp`

There's a race between 2 and 3 where something could create a file

Fortunately it _doesn't really matter_ since it's just a test.